tests/runtest.py

```diff
@@ -32,34 +32,33 @@
 import time
 import subprocess
 
-if len(sys.argv) not in {4, 5, 6, 7, 8, 9}:
-    print("""Usage: %s input_file logfile top_srcdir doreaptest alt_output_file alt_psi4_exe alt_psi4datadir""" % (sys.argv[0]))
+if len(sys.argv) not in {4, 5, 6, 7, 8}:
```

Comment:
It might be good to eventually `argparse` this if the arguments here will change in the future.

Reply:
Agree that argparse is the sensible way. But this is called with fixed argument list from CMake, so it doesn't really need to be helpful or flexible. Another possibility is to replace the whole fn by a call to qcengine.execute.